### PR TITLE
add additional ebooks from accordbox

### PR DESCRIPTION
**Pull request by lb-**

### CODE REVIEW

**Review by cnk — APPROVED** [review]
These are excellent resources. Yes please add them.

**Review by cnk — APPROVED** [review]

**Comment by lb-** [comment]
Thanks @cnk but I think you may not have write access to this repo - maybe chat on the core team about getting access. Otherwise will just wait for another person to review.

And yeah - they are good resources.

**Review by thibaudcolas — APPROVED** [review]

**Pull request merged**
